JASP-Engine/JASP/R/ldLogistic.R

```diff
@@ -0,0 +1,200 @@
+  .ldFillCDFContainer(cdfContainer, options, .ldFormulalogisticCDF)
+  
+  qfContainer  <- .ldGetPlotContainer(jaspResults, options, "plotQF", gettext("Quantile Function"), 5)
+  .ldFillQFContainer(qfContainer,   options, .ldFormulaLogisticQF)
+  
+  #### Generate and Display data section ----
+  # simulate and read data
+  .simulateData(jaspResults, options)
+  
+  ready <- options[['variable']] != ""
+  errors <- FALSE
+  if(ready && is.null(dataset)){
+    dataset <- .readDataSetToEnd(columns.as.numeric = options[['variable']])
+    
+    variable <- dataset[[.v(options[['variable']])]]
+    variable <- variable[!is.na(variable)]
+    errors <- .hasErrors(dataset, type = c("observations", "variance", "infinity", "limits"),
+                         observations.amount = "<2",
+                         limits.min = options$support$min, limits.max = options$support$max, 
+                         exitAnalysisIfErrors = FALSE)
+  }
+  
+  # overview of the data
+  dataContainer <- .ldGetDataContainer(jaspResults, options, errors)
+  
+  readyDesc <- ready && (isFALSE(errors) || (is.null(errors$infinity) && is.null(errors$observations)))
+  .ldSummaryContinuousTableMain(dataContainer, variable, options, readyDesc)
+  .ldObservedMomentsTableMain  (dataContainer, variable, options, readyDesc)
+  .ldPlotHistogram             (dataContainer, variable, options, readyDesc)
+  .ldPlotECDF                  (dataContainer, variable, options, readyDesc)
+  
+  
+  #### Fit data and assess fit ----
+  
+  readyFit <- ready && isFALSE(errors)
+  #### Maximum Likelihood ----
+  if(options$methodMLE){
+    mleContainer <- .ldGetFitContainer(jaspResults, options, "mleContainer", "Maximum likelihood", 7, errors)
+    
+    # parameter estimates
+    mleEstimatesTable  <- .ldEstimatesTable(mleContainer, options, TRUE, TRUE, "methodMLE")
+    mleResults   <- .ldMLEResults(mleContainer, variable, options, readyFit, options$distNameInR)
+    .ldFillLogisticEstimatesTable(mleEstimatesTable, mleResults, options, readyFit)
+    
+    
+    # fit assessment
+    mleFitContainer    <- .ldGetFitContainer(mleContainer, options, "mleFitAssessment", "Fit Assessment", 8)
+    
+    # fit statistics
+    mleFitStatistics   <- .ldFitStatisticsTable(mleFitContainer, options, "methodMLE")
+    mleFitStatisticsResults <- .ldFitStatisticsResults(mleContainer, mleResults$fitdist, variable, options, readyFit)
+    .ldFillFitStatisticsTable(mleFitStatistics, mleFitStatisticsResults, options, readyFit)
+    # fit plots
+    .ldFitPlots(mleFitContainer, mleResults$fitdist$estimate, options, variable, readyFit)
+    
+  }
+  
+  return()
+}
+
+.recodeOptionsLDLogistic <- function(options){
+  options[['parValNames']] <- c("mu", "sigma")
+  
+  options[['pars']]   <- list(location = options[['mu']], scale = options[['sigma']])
+  options[['pdfFun']] <- dlogis
+  options[['cdfFun']] <- plogis
+  options[['qFun']]   <- qlogis
+  options[['rFun']]   <- rlogis
```

Comment:
```suggestion
  options[['pdfFun']] <- stats::dlogis
  options[['cdfFun']] <- stats::plogis
  options[['qFun']]   <- stats::qlogis
  options[['rFun']]   <- stats::rlogis
```
a little bit pedantic, but when we switch to modules we'll either need to prepend all of these with `stats::` or otherwise we need to import in the namespace (using roxygen2, ideally). Doing it now already might alleviate that step. If you prefer doing this with roxygen2 you can ignore this comment :)